src/main/java/ab/common/lib/register/RecipeListAB.java

```diff
@@ -791,6 +761,90 @@ public static void setupCrafting() {
+                "Nebula",
+
+                new ItemStack(ItemListAB.itemNebulaHelm),
+                12,
+                new AspectList().add(Aspect.EARTH, 128).add(Aspect.MAGIC, 64).add(Aspect.CRYSTAL, 20)
+                        .add(Aspect.METAL, 20).add(Aspect.SENSES, 20).add(Aspect.ELDRITCH, 20),
+                new ItemStack(ModItems.gaiaHead),
+                new ItemStack[] { new ItemStack(ModItems.elementiumHelm), new ItemStack(ModItems.laputaShard, 1, 15),
+                        OreDictionary.getOres("gemExquisiteBotaniaDragonstone").get(0),
+                        new ItemStack(ItemListAB.itemABResource, 1, 5), new ItemStack(ItemListAB.itemNebulaRing),
+                        new ItemStack(ItemListAB.itemABResource, 1, 5), new ItemStack(ItemListAB.itemABResource, 1, 3),
+                        new ItemStack((Item) Item.itemRegistry.getObject("Thaumcraft:ItemAmuletRunic"), 1, 1),
+                        new ItemStack(ModItems.terrasteelHelm),
+                        new ItemStack((Item) Item.itemRegistry.getObject("Thaumcraft:ItemAmuletRunic"), 1, 1),
+                        new ItemStack(ItemListAB.itemABResource, 1, 3), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        new ItemStack(ItemListAB.itemNebulaRing), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        OreDictionary.getOres("gemExquisiteBotaniaDragonstone").get(0),
+                        new ItemStack(ModItems.laputaShard, 1, 15), });
+
+        NebulaChest = ThaumcraftApi.addInfusionCraftingRecipe(
+                "Nebula",
+
+                new ItemStack(ItemListAB.itemNebulaChest),
+                12,
+                new AspectList().add(Aspect.EARTH, 128).add(Aspect.MAGIC, 64).add(Aspect.CRYSTAL, 20)
+                        .add(Aspect.METAL, 20).add(Aspect.SENSES, 20).add(Aspect.FLIGHT, 20),
+                new ItemStack(ModItems.flightTiara),
+                new ItemStack[] { new ItemStack(ModItems.elementiumChest), new ItemStack(ModItems.laputaShard, 1, 15),
+                        new ItemStack(ModBlocks.storage, 1, 2), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        new ItemStack(ItemListAB.itemNebulaRing), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        new ItemStack(BlockListAB.blockABStorage, 1, 0),
+                        new ItemStack((Item) Item.itemRegistry.getObject("Thaumcraft:ItemAmuletRunic"), 1, 1),
+                        new ItemStack(ModItems.terrasteelChest),
+                        new ItemStack((Item) Item.itemRegistry.getObject("Thaumcraft:ItemAmuletRunic"), 1, 1),
+                        new ItemStack(BlockListAB.blockABStorage, 1, 0), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        new ItemStack(ItemListAB.itemNebulaRing), new ItemStack(ItemListAB.itemABResource, 1, 5),
+                        new ItemStack(ModBlocks.storage, 1, 2), new ItemStack(ModItems.laputaShard, 1, 15),
```

Comment:
ModBlocks.storage must be replaced with oredictionary for GT material compatibility.

Reply:
The modBlocks.storage are only from botania, so no need of oredic it, if you mean to place something like this is okay for me:
 new ItemStack(ModBlocks.storage, 1, Constants.STORAGE_META_MANASTEELBLOCK),

Reply:
Done

Reply:
GT materials mod generates blocks, plates and all other things, and while it *currently* doesn't create doubles where the native botania version exists, it has done so for plenty of other mods and therefore that's not a good assumption to make.